### PR TITLE
Remove slider plugin and replace with custom scroll

### PR DESCRIPTION
**Pull request by ukirderohit**

### CODE REVIEW

File: packages/im-components/src/molecules/filmCard/filmcard.module.scss [hunk 1]
```diff
@@ -1,55 +1,44 @@
...
+  transition: all 0.3s ease;
 
-.filmcard:hover {
-  -webkit-transform: translateY(-0.5rem);
-  transform: translateY(-0.5rem);
-}
+  .relative {
+    position: relative;
+    .details {
+      position: absolute;
```
Comment (pranay2401) [inline packages/im-components/src/molecules/filmCard/filmcard.module.scss:8]: add line spacing between css classes

File: packages/im-ott/src/organisms/filmList/FilmList.js [hunk 2]
```diff
@@ -85,13 +101,27 @@ const FilmList = (props) => {
   }
 
   return (
-    <div className={styles.horizontalList}>
+    <div className={styles.mt1}>
```
Comment (pranay2401) [inline packages/im-ott/src/organisms/filmList/FilmList.js:104]: should have a more meaningful name

File: packages/im-components/src/molecules/filmCard/filmcard.module.scss [hunk 3]
```diff
@@ -1,55 +1,44 @@
...
-    rgba(0, 0, 0, 1) 100%
-  );
+  .filmCardImgFeatured {
+    max-height: 45rem;
+    object-fit: contain;
+    margin-right: 0.8rem;
+    width: calc(45rem * (9 / 16));
+    border-radius: 1rem;
+  }
+  &:hover {
```
Comment (pranay2401) [inline packages/im-components/src/molecules/filmCard/filmcard.module.scss:45]: add line space

File: packages/im-components/src/molecules/filmCard/filmcard.module.scss [hunk 4]
```diff
@@ -1,55 +1,44 @@
...
-}
+  transition: all 0.3s ease;
 
-.filmcard:hover {
-  -webkit-transform: translateY(-0.5rem);
-  transform: translateY(-0.5rem);
-}
+  .relative {
+    position: relative;
+    .details {
```
Comment (pranay2401) [inline packages/im-components/src/molecules/filmCard/filmcard.module.scss:7]: add line spacing

**Review by pranay2401 — CHANGES_REQUESTED** [review]
Please address comments

File: packages/im-ott/src/organisms/filmList/FilmList.js (hunk 2 above)
Comment (ukirderohit) [reply packages/im-ott/src/organisms/filmList/FilmList.js:104]: most class names are like these if you check any library, its standard name for mt-1 margin-top: 0.5rem

**Review by pranay2401 — APPROVED** [review]

**Pull request merged**